vms/platformvm/vm.go

```diff
@@ -143,16 +127,21 @@ func (vm *VM) Initialize(
 	}
 
 	rewards := reward.NewCalculator(vm.RewardConfig)
+
+	execConfig, err := config.GetExecutionConfig(configBytes)
```

Comment:
nit: I'd move it up where it was. No point instantiating anything if we can't parse the config.